### DISCUSSION TITLE
Temps d’attente avant de Ping combat

### DISCUSSION THREAD

**Discussion by Voisin1234** [Crownicles/Crownicles#2374] [Suggestions fermées]
Bonjour, j'ai constaté plusieurs abus un peu relou sur les Ping pour les mentions combat. Actuellement, certains ping toutes les 10 minutes quand ils n'ont pas trouvé de combat, et dans tous les chanels. Je voudrais faire une proposition qu'il soit possible de ping une seul fois toutes les 30 minutes voir 1 heures. Il pourrait y avoir des exceptions bien sûr si on vient de Ping et qu'on vien de combattre mais qu'entre-temps si on vient de récupérer toute notre vie ou autre là il est normal de pouvoir ping une autre fois. je propose de mettre des sanctions aux personnes qui ne respectent pas la règle.

**Comment by BastLast** [comment]
Yo ici c'est pour les suggestions pr le jeu, merci de mentionner les staff sur discord pour qu'ils s'occupent de ce problème

**Reply by Voisin1234** [reply]
Mdr
